mindmeld/augmentation.py

```diff
@@ -359,9 +546,25 @@ def _translate(self, *, queries, model, tokenizer, **kwargs):
             all_translated_queries.extend(translated_queries)
         return all_translated_queries
 
+    def _prepare_inputs(self, queries):
+        """Removes any markdown formatting in the query
+
+            Args:
+                queries (list(str)): List of queries to be paraphrased
+
+            Returns:
+                unannotated queries (list(str))
+
+        """
+        processed_queries = []
+        for query in queries:
+            processed_query = load_query(query)
+            processed_queries.append(processed_query.query.text.strip())
+        return processed_queries
```

Comment:
It's a bit unclear here that 'processed_queries' refers to a list of 'str' objects and not 'ProcessedQuery' objects. Perhaps consider renaming this to 'processed_query_texts'?